oracle-plugin/widgets/Oracle-batchsink.json

```diff
@@ -39,9 +39,28 @@
             "default": "1521"
           }
         },
+        {
+          "name": "connectionType",
+          "label": "SID/Service Name",
```

Comment:
It looks weird if there are two properties with the same label. Do you know if there is some term that most people use for this? Do you think 'Connection Type' is hard to understand?

Reply:
Agree, "Connection Type" is clear to understand